### PR TITLE
Added server entry for ICQ-Chat network to servlist.c

### PR DESCRIPTION
**Pull request by haziegler**
This adds a server entry for the ICQ-Chat IRC network (See https://icq-chat.com)

Note that this server entry is _not_ the discontinued IRC service that was provided by icq.com several years ago.

### CODE REVIEW

File: src/common/servlist.c [hunk 1]
```diff
@@ -201,6 +201,9 @@ static const struct defaultserver def[] =
 	{"Hashmark",	0},
 	{0,			"irc.hashmark.net"},
 
+	{"ICQ-Chat", 0},
```
Comment (TingPing) [inline src/common/servlist.c:204]: Update this to include that it supports SASL and SSL like others.

**Review by TingPing — CHANGES_REQUESTED** [review]

File: src/common/servlist.c (hunk 1 above)
Comment (haziegler) [reply src/common/servlist.c]: Updated

**Comment by haziegler** [comment]
I've updated the server entry.

File: src/common/servlist.c [hunk 2]
```diff
@@ -207,6 +207,9 @@ static const struct defaultserver def[] =
 	{"Hashmark",	0},
 	{0,			"irc.hashmark.net"},
 
+	{"ICQ-Chat", 0, 0, 0, LOGIN_SASL, 0, TRUE},
+	{0,			"irc.icq-chat.com/+6697"},
```
Comment (TingPing) [inline src/common/servlist.c:211]:
```suggestion
	{0,			"irc.icq-chat.com"},
```

**Comment by haziegler** [comment]
@TingPing is there anything else needed to get this merged?

**Comment by TingPing** [comment]
Somebody  else did a mass update of the list, sorry forgot to merge this earlier,  so this needs rebased.

**Pull request closed without merging**